CedarJava/src/test/java/com/cedarpolicy/SharedIntegrationTests.java

```diff
@@ -176,25 +176,25 @@ private static class JsonEntity {
      * files in this array will be executed as integration tests.
      */
     private static final String[] JSON_TEST_FILES = {
-//        "tests/example_use_cases_doc/1a.json",
-//        "tests/example_use_cases_doc/2a.json",
-//        "tests/example_use_cases_doc/2b.json",
-//        "tests/example_use_cases_doc/2c.json",
-//        "tests/example_use_cases_doc/3a.json",
-//        "tests/example_use_cases_doc/3b.json",
-//        "tests/example_use_cases_doc/3c.json",
+        "tests/example_use_cases_doc/1a.json",
+        "tests/example_use_cases_doc/2a.json",
+        "tests/example_use_cases_doc/2b.json",
+        "tests/example_use_cases_doc/2c.json",
+        "tests/example_use_cases_doc/3a.json",
+        "tests/example_use_cases_doc/3b.json",
+        "tests/example_use_cases_doc/3c.json",
         // "tests/example_use_cases_doc/4a.json", // currently disabled due to action attributes
         // "tests/example_use_cases_doc/4c.json", // currently disabled due to action attributes
         // "tests/example_use_cases_doc/4d.json", // currently disabled due to action attributes
         // "tests/example_use_cases_doc/4e.json", // currently disabled due to action attributes
         // "tests/example_use_cases_doc/4f.json", // currently disabled due to action attributes
         // "tests/example_use_cases_doc/5b.json", // currently disabled due to action attributes
         // Need to change extension handling to match natural JSON CRs
-//        "tests/ip/1.json",
-//        "tests/ip/2.json",
-//        "tests/ip/3.json",
-//        "tests/multi/1.json",
-//        "tests/multi/2.json",
+        "tests/ip/1.json",
+        "tests/ip/2.json",
+        "tests/ip/3.json",
+        "tests/multi/1.json",
+        "tests/multi/2.json",
```

Comment:
Although these lines are uncommented, the ones below that actually run the tests (L220-242) still appear to be commented out :)

Reply:
I think these tests are run. The commented lines are for the corpus tests, which still have some failures.

Reply:
Good point. I shouldn't try to review things on Friday evenings...